tools/accuracy_checker/accuracy_checker/adapters/yolo.py

```diff
@@ -798,6 +798,70 @@ def xywh2xyxy(x):
     return y
 
 
+class YoloxsAdapter(Adapter):
+    __provider__ = 'yoloxs'
+    prediction_types = (DetectionPrediction, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'score_threshold': NumberField(value_type=float, optional=True, min_value=0, default=0.001,
+                                     description="Minimal accepted score value for valid detections."),
+            'box_format_xywh': BoolField(optional=True, default=False,
+                                         description="Indicates that box output format is xywh.")
+        })
+        return parameters
+
+    def configure(self):
+        self.score_threshold = self.get_value_from_config('score_threshold')
+        self.box_format_xywh = self.get_value_from_config('box_format_xywh')
+
+    def process(self, raw, identifiers, frame_meta):
+        result = []
+        raw_outputs = self._extract_predictions(raw, frame_meta)
+
+        num_classes = 0
+        x_mins, y_mins, x_maxs, y_maxs = [], [], [], []
+
+        for identifier, meta in zip(identifiers, frame_meta):
+            if len(self.additional_output_mapping) > 0:
+                boxes = np.array(raw_outputs[self.additional_output_mapping['boxes']]).squeeze()
```

Comment:
is it possible to provide explicit config fields for required output layer names specification?

Reply:
Additional config fields have been added.